src/main/java/catan/services/util/random/RandomUtil.java

```diff
@@ -39,12 +37,17 @@ public HexType pullRandomHexType(List<HexType> possibleHexTypes) {
     }
 
     //TODO: add rule to avoid placing 6 and 8 dice numbers close to each other
-    public int pullRandomDiceNumber(List<Integer> possibleDiceNumbers) {
+    public int pullRandomHexDiceNumber(List<Integer> possibleDiceNumbers) {
         int randomDiceNumber = (int) (rvg.randomValue() * possibleDiceNumbers.size());
 
         return possibleDiceNumbers.remove(randomDiceNumber);
     }
 
+    public int getRandomDiceNumber() {
+        int diceNumber = (int) Math.ceil(rvg.randomValue() * 6);
+        return diceNumber == 0 ? 1 : diceNumber;
```

Comment:
in such case chance of value 1 is two-times more than all other values, 6 has the smallest chance to be returned